test/src/components/Button-test.js

```diff
@@ -26,4 +26,9 @@ describe('Button', () => {
     React.addons.TestUtils.Simulate.click(node);
     customHandleClick.called.should.be.true;
   });
+  it('should take in a color attribute', () => {
+    let coloredButton = render(<Button color={'teal'}>Submit</Button>).findTag('button');
+    coloredButton.props.className.should.contain('teal');
+    coloredButton.getDOMNode().getAttribute('class').should.contain('teal');
+  });
```

Comment:
Instead of `findTag()`, use `findClass('teal')` this will find the element and do the assertion.  It also removes the dependency of this test on the structure of the button.  Right now, the test will fail if the className is not found on the first on the root element of the button (props.className).  With `findClass()` it will work for any structure and only fail if the class name is missing.